product_docs/docs/hadoop_data_adapter/2/07_features_of_hdfs_fdw.mdx

```diff
@@ -24,6 +24,18 @@ Hadoop Foreign Data Wrapper supports aggregate pushdown. It pushes the aggregate
 
 For more information, see [Example: Aggregate pushdown](10b_example_aggregate_pushdown).
 
+## ORDER BY pushdown
+
+Hadoop Foreign Data Wrapper supports order by push-down. If possible, push `ORDER BY` clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. 
+
+- [Example: ORDER BY pushdown](10c_example_order_by_push_down)
```

Comment:
You should avoid single item bulleted lists. Use the same format as the other sections in the topic: For more information, see [Example: Aggregate pushdown](https://deploy-preview-3331--edb-docs-staging.netlify.app/docs/hadoop_data_adapter/latest/10b_example_aggregate_pushdown/).

Reply:
changes done as per both comments